apps/etterlatte-saksbehandling-ui/client/src/components/person/brev/NyttBrevModal.tsx

```diff
@@ -61,36 +61,48 @@ export const NyttBrevModal = ({ sakId }: { sakId: number }) => {
 
         <form onSubmit={handleSubmit(opprettBrev)}>
           <Modal.Body>
-            <FlexRow $spacing>
+            <VStack gap="4">
               <Select
                 error={errors?.type?.message}
                 label="Type"
                 {...register('type', {
                   required: { value: true, message: 'Feltet er påkrevd' },
                 })}
               >
-                <option value="OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_VARSELBREV">Varselbrev om aktivitetsplikt</option>
+                <option value="OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_INFORMASJON_4MND">
+                  Informasjon om aktivitetsplikt ved 4 måneder
+                </option>
```

Comment:
Noe jeg kom på nå (som kanskje passer best i en annen pr) er at denne burde være tilgjengelig kun for OMS-saker, og dermed filtreres bort hvis det er en barnepensjonssak